data/examples/rasa/multiple_files_json/demo-rasa-affirm.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Can we please move these into `data/test/multiple_files_json` instead of `data/examples/rasa/multiple_files_json`? (same for the md)

Reply:
Not sure. All the files in this test live in `data/examples` so the multi-file examples/test would be the only ones that are not there. Even worse, the reference cases `data/examples/rasa/demo-rasa.{md,json}` would probably need to stay in the examples directory while the multi-file example would move to `data/test`...
Or do you want me to move **all** the test files to `data/test`?
I would prefer a consistent solution. ;-)

Reply:
I see your point and I agree with you, a consistent solution would be best. The files in `/data/examples` are meant to guide the user with respect to the file formats and serve as examples on how the format works. (so the `demo-rasa-noents.json` shouldn't be in there as it doesn't add any value but is just used for a test)

For now I'd still only move the `multiple_files_json` as they are only used for testing and hence should go into that folder - but as said, I agree with you we should think about moving the other rasa examples to the test folder as well and update the links to that file.